.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -0,0 +1,98 @@
+name: 🐛 Report a bug
+description: Any errors.
+labels: ["needs triage", "bug"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Before you go any further. Is this really a **🐛 bug**?
+
+        If it's a question about how al-folio works, have a look at our [documentation](../../README.md),
+        [frequently asked questions](../../FAQ.md), [past questions](https://github.com/alshedivat/al-folio/discussions/categories/q-a),
+        or [ask a question](https://github.com/alshedivat/al-folio/discussions/new?category=q-a).
+
+  - type: checkboxes
+    id: requirements
+    attributes:
+      label: Have you checked that your issue isn't already filed?
+      description: >
+        Please check if somebody else has already filed the same issue.
+        If you find a similar issue, please add a 👍 reaction or comment on the original post.
+      options:
+        - label: I read through [FAQ](../../FAQ.md) and searched through the [past issues](https://github.com/alshedivat/al-folio/issues), none of which addressed my issue.
+          required: true
+        - label: Yes, I have checked that this issue isn't already filed.
+          required: true
+
+  - type: input
+    attributes:
+      label: Bug description
+      description: A description of the 🐛 bug.
+      placeholder: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: How to reproduce the bug
+      description: Provide steps to reproduce the 🐛 bug.
+      placeholder: |
+        Include steps to reproduce, the expected behaviour, and the actual behaviour.
+
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Error messages and logs
+      description: >
+        Provide any error messages and/or logs
+      placeholder: "Copy the complete error messages and logs"
+      value: |
+        ```
+        The error message you got, with the full traceback if available. Please paste it between these triple backticks.
+        ```
+    validations:
+      required: false
+
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Where are you seeing the problem on?
+      description: select all environments where you have experienced this issue
+      multiple: true
+      options:
+        - "Running locally with Docker"
+        - "Running locally without Docker"
```

Comment:
if running locally w/o docker, it'd be good to ask about jekyll and ruby versions